src/quizzes/chapter-2/2.2/advanced-workflow-quiz.js

```diff
@@ -0,0 +1,45 @@
+const rawQuizdown = `
+---
+shuffleQuestions: true
+shuffleAnswers: true
+---
+
+# What are primary benefits of using self-hosted runners in GitHub Actions? Select all that apply
+For help on self-hosted runners, check out the [GitHub docs](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners#differences-between-github-hosted-and-self-hosted-runners)
+
+- [x] Improved security due to the isolation of workflows on dedicated infrastructure
+	> Good. With self-hosted runners, you control where they are hosted and run from, which means you can fine-tune the security to your needs
+- [x] Increased control over hardware resources, software environments and build tools
+	> Good. You can modify self-hosted runners to have different package dependencies/versions and build tools than the standard GitHub runners. Sometimes this is required for a project's success if no GitHub runner offers the environment you need
+- [ ] Increased reliability as GitHub manages and monitors self-hosted runners 24/7
```

Comment:
This reads weird to me. It is like a pro for using GitHub hosted runners not selfhosted. I know it is not correct but maybe it should be worded like, "Increased reliability as you are not dependent on GitHub runners which can go down" or something.

Reply:
At that point it would switch to be a correct answer. The "gotcha" in this line is that it says that GitHub manages and monitors self-hosted runners, which it does not.

Reply:
I'm open to switching this in the future if you still don't like it. I was just trying to make trick answers.